tests/test_e2e.py

```diff
@@ -1,37 +1,66 @@
-import subprocess
 import pytest
+import subprocess
+import tempfile
+import yaml
 from kubernetes import client, config, utils
+from tests.utils import wait_for_deployments_ready
 
+E2E_INPUT_MANIFEST = "e2e-manifests/{}.yml"
+E2E_EXPECTED_YAML = "e2e-manifests/expected/{}.yml"
+E2E_RUNNER_IMAGE = "localhost:5000/illuminatio-runner:dev"
 
-@pytest.mark.e2e
-def test_deny_all_traffic_to_an_application():
-    namespace = "01-deny-all"
+
+@pytest.fixture
+def kubernetes_utils():
     config.load_kube_config()
     k8s_client = client.ApiClient()
-    corev1 = client.CoreV1Api()
+    return k8s_client, client.CoreV1Api(), client.AppsV1Api()
```

Comment:
Actually I think it's fine to only return `k8s_client` because only `k8s_client` and `apps_v1` are used or `core_v1` but never all three of them.

Reply:
I simplified it to have one fixture per client type which requests the fixture for loading the config